src/conversion_helper.js

```diff
@@ -522,6 +522,37 @@ function shouldConvertComment(comment, regexArray = globalIgnore, shouldBeUnique
 }
 
 /*
+  Input: Comment
+    "1 3/4 miles"
+  Output: Comment with body processed to fix user formatting
+    "1.75 miles"
+*/
+function preprocessComment(comment) {
+  function fractionProcessor(input) {
+    var mixedRegex = new RegExp((rh.numberRegex + /(?:[\s+-]+)/.source + rh.numberRegex + "/" + rh.numberRegex + /([\s-])/.source), 'gi');
+    var fractionRegex = new RegExp(rh.numberRegex + "/" + rh.numberRegex + /([\s-])/.source, 'gi');
```

Comment:
The line `rh.numberRegex + "/" + rh.numberRegex + /([\s-])` is repeated twice (once in mixed regex and once here), if we change the order of the variable declarations, we could re-use `fractionRegex` within `mixedRegex`. Also, minor stylistic point, can you use "const" instead of "var"?